### PR TITLE
Port sale_quotation_number to v8

### PR DESCRIPTION
**Pull request by franksl**
Hope it helps

### CODE REVIEW

**Comment by eLBati** [comment]
:+1: thanks

**Comment by jgrandguillaume** [comment]
:+1:  Thanks

File: sale_quotation_number/model/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        if default is None:
+            default = {}
+        default['name'] = '/'
+        return super(SaleOrder, self).copy(default=default)
+
+    @api.model
+    def create(self, vals):
+        if vals.get('name', '/') == '/':
+            vals['name'] = self.env['ir.sequence'].next_by_code(
+                'sale.quotation') or '/'
```
Comment (nbessi) [inline sale_quotation_number/model/sale_order.py:39]: it is posssible should'nt we raise an error?

**Comment by nbessi** [comment]
Please revert the pr you do not respect the OCA rules.

three thumb up of two and 5 days of delay

**Comment by nbessi** [comment]
my baad. I have miss read commit time and PR data sorry

**Pull request merged**